### PR TITLE
feat: Create new environment variable to toggle directpath scoped to cloud bigtable.

### PR DESCRIPTION
**Pull request by meeral-k**
feat: Create new environment variable to toggle directpath scoped to cloud bigtable.

### CODE REVIEW

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java [hunk 1]
```diff
@@ -100,6 +100,7 @@ public class EnhancedBigtableStubSettings extends StubSettings<EnhancedBigtableS
   private static final int MAX_MESSAGE_SIZE = 256 * 1024 * 1024;
   private static final String SERVER_DEFAULT_APP_PROFILE_ID = "";
 
+  private static final String CBT_ENABLE_DIRECTPATH = "CBT_ENABLE_DIRECTPATH";
```
Comment (igorbernstein2) [inline google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java:103]: Please add a TODO to document this

Also, if this is will be canonical way to enable DirectPath please update integration tests to use this mechanism:
- update directpath test profiles in the pom to set this env var
  - https://github.com/googleapis/java-bigtable/blob/e7f03fc7d252a7ff6c76a8e6e0a9e6ad3dcbd9d5/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java#L208-L210
  -  https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/pom.xml#L554-L590  
- remove the manual way to enable it: https://github.com/googleapis/java-bigtable/blob/e7f03fc7d252a7ff6c76a8e6e0a9e6ad3dcbd9d5/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java#L208-L210

Comment (meeral-k) [reply google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java:104]: done

Comment (igorbernstein2) [reply google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java:104]: can you tag the todo with your github handle?

File: google-cloud-bigtable/pom.xml [hunk 2]
```diff
@@ -498,6 +498,7 @@
                     <bigtable.enable-grpc-logs>${bigtable.enable-grpc-logs}</bigtable.enable-grpc-logs>
                     <bigtable.grpc-log-dir>${project.build.directory}/test-grpc-logs/directpath-it</bigtable.grpc-log-dir>
                     <bigtable.connection-mode>REQUIRE_DIRECT_PATH</bigtable.connection-mode>
+                    <CBT_ENABLE_DIRECTPATH>true</CBT_ENABLE_DIRECTPATH>
```
Comment (igorbernstein2) [inline google-cloud-bigtable/pom.xml:501]: I think you want to set an environment variable not a system property.

Comment (meeral-k) [reply google-cloud-bigtable/pom.xml]: fixed and verified

File: google-cloud-bigtable/pom.xml [hunk 3]
```diff
@@ -499,6 +499,9 @@
                     <bigtable.grpc-log-dir>${project.build.directory}/test-grpc-logs/directpath-it</bigtable.grpc-log-dir>
                     <bigtable.connection-mode>REQUIRE_DIRECT_PATH</bigtable.connection-mode>
                   </systemPropertyVariables>
+                  <environmentVariables>
+                    <CBT_ENABLE_DIRECTPATH>true</CBT_ENABLE_DIRECTPATH>
+                  </environmentVariables>
```
Comment (igorbernstein2) [inline google-cloud-bigtable/pom.xml:504]: please add this to bigtable-directpath-ipv4only-it profile as well

File: google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java (hunk 1 above)
Comment (meeral-k) [reply google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java:104]: done

File: google-cloud-bigtable/pom.xml (hunk 3 above)
Comment (meeral-k) [reply google-cloud-bigtable/pom.xml:504]: done

**Review by igorbernstein2 — APPROVED** [review]

**Comment by meeral-k** [comment]
```
[11:55:20 PDT][ERROR] Timed out waiting for GitHub to return mergeable status for the pull (PR). GitHub is taking too long to compute whether this PR is mergeable. Please retry the build in a few minutes. You can do this by attaching a kokoro:force-run label on your PR.
```

the tests need to be re-run. I don't see an option to relabel this PR with kokoro:force-run

**Pull request merged**